scripts/fix-quotes.sh

```
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Script to fix quote style issues in JavaScript files
+cd "$(dirname "$0")/.." || exit 1
+
+# Detect the platform to use appropriate sed command
+if [[ "$(uname)" == "Darwin" ]]; then
+    # macOS (BSD sed)
+    sed_cmd="sed -i ''"
+else
+    # Linux/Unix (GNU sed)
+    sed_cmd="sed -i"
```

Shellcheck (0.10.0)

[warning] sed_cmd appears unused. Verify use (or export if used externally).

(SC2034)